packages/helper-react/vite.config.ts

```diff
@@ -0,0 +1,27 @@
+import { PluginOption, defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+import dts from 'vite-plugin-dts'
+import { resolve } from 'path'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  // there is a typing problem with the dts package
+  plugins: [dts() as unknown as PluginOption, react()],
+  build: {
+    lib: {
+      entry: resolve(__dirname, 'src/index.ts'),
+      name: 'react-helper',
+      fileName: 'index',
+    },
+    rollupOptions: {
+      external: ['react', 'react-dom', '@storyblok/field-plugin'],
+      output: {
+        globals: {
+          react: 'React',
+          'react-dom': 'ReactDOM',
+          '@storyblok/field-plugin': 'FieldPlugin',
+        },
+      },
```

Comment:
Although we're depending on these 3 libraries, we're excluding them in the final bundle output.

Reply:
I'll show you this on Wednesday. Don't forget to ask! :)